load/UpdateDBPLPTSTable.py

```diff
@@ -339,26 +357,34 @@ def updateBibleFilesetCopyrightOrganizations(self, filesetList):
 		#unknownCopyrights = orgs.validateCopyrights()
 		orgs.update_licensors(filesetList)
 
-	def upsertBibleFileset(self, dbConn, setTypeCode, setSizeCode, filesetId, isContentLoaded=0):
+	def upsertBibleFileset(self, dbConn, setTypeCode, setSizeCode, filesetId, isContentLoaded=0, isArchived=0):
 		tableName = "bible_filesets"
 		pkeyNames = ("hash_id",)
-		attrNames = ("id", "asset_id", "set_type_code", "set_size_code", "content_loaded")
+		attrNames = ("id", "asset_id", "set_type_code", "set_size_code", "content_loaded", "archived")
 		updateRows = []
 		bucket = self.config.s3_vid_bucket if setTypeCode == "video_stream" else self.config.s3_bucket	
 		hashId = self.getHashId(bucket, filesetId, setTypeCode)
-		row = dbConn.selectRow("SELECT id, asset_id, set_type_code, set_size_code, content_loaded FROM bible_filesets WHERE hash_id=%s", (hashId,))
+		row = dbConn.selectRow("SELECT id, asset_id, set_type_code, set_size_code, content_loaded, archived FROM bible_filesets WHERE hash_id=%s", (hashId,))
```

Comment:
The logic below is getting a bit complicated.
I propose: 
1) return after the row=None check
2) then add a check to ensure that upsertBibleFileset is not called with both isContentLoaded = 1 and isArchived = 1
3) then process isContentLoaded = 1 and return
4) then process isArchived = 1
then

Reply:
@bradflood , This method has been updated to simplify its logic. You can check it from this line: https://github.com/faithcomesbyhearing/dbp-etl/pull/140/files#diff-a8b84dcb2a008d41509d439d0dad3855517bc429e17563a012d78d868ced3024R361